src/modules/colorPicker/ColorPickerUI/Helpers/ColorHelper.cs

```diff
@@ -58,7 +58,12 @@ internal static (double red, double green, double blue) ConvertToDouble(Color co
         /// <param name="color">The <see cref="Color"/> to convert</param>
         /// <returns>The hue [0°..360°], saturation [0..1] and brightness [0..1] of the converted color</returns>
         internal static (double hue, double saturation, double brightness) ConvertToHSBColor(Color color)
-            => (color.GetHue(), color.GetSaturation(), color.GetBrightness());
+        {
+            var min = Math.Min(Math.Min(color.R, color.G), color.B) / 255d;
+            var max = Math.Max(Math.Max(color.R, color.G), color.B) / 255d;
+
+            return (color.GetHue(), max == 0d ? 0d : (max - min) / max, max);
```

Comment:
Since the value is supposed to be the same as HSV, why not call "ConvertToHSVColor" from here instead, to avoid code duplication? Looks like the same code to me.